### PR TITLE
using zion-ui instead of the individual ui components

### PR DESCRIPTION
**Pull request by joeycozza**

### CODE REVIEW

**Comment by spacerockzero** [comment]
oooh. this is a nice change. i welcome it

File: packages/react-scripts/scripts/utils/frontierInit.js [hunk 1]
```diff
@@ -58,12 +58,7 @@ function installFrontierDependencies(appPath, appName, answers, ownPath) {
...
     ...[
       '@emotion/core@10',
-      '@fs/zion-avatar',
       '@fs/zion-axios',
-      '@fs/zion-button',
-      '@fs/zion-card',
-      '@fs/zion-chip',
-      '@fs/zion-dialog',
       '@fs/zion-frontend-friends',
       '@fs/zion-grid',
```
Comment (cadebward) [inline packages/react-scripts/scripts/utils/frontierInit.js]: need to remove grid from here, too

File: packages/react-scripts/scripts/utils/frontierInit.js [hunk 2]
```diff
@@ -73,8 +68,7 @@ function installFrontierDependencies(appPath, appName, answers, ownPath) {
       '@fs/zion-subnav',
       '@fs/zion-theme',
```
Comment (cadebward) [inline packages/react-scripts/scripts/utils/frontierInit.js]: theme is also in zion-ui

Comment (cadebward) [reply packages/react-scripts/scripts/utils/frontierInit.js]: looks like theme is also being installed down in dev deps?

**Pull request merged**